Makefile

```diff
@@ -16,27 +16,16 @@
 ## Copyright (C) 2016  Dirk Eddelbuettel
 ## and released under GPL (>=2 )
 
-
-## check with https://github.com/adobe-fonts/source-code-pro/releases for version
-srcfile1=https://github.com/mozilla/Fira/archive/4.202.tar.gz
-outfile1=$(shell basename ${srcfile1})
-srcfile2=https://github.com/mozilla/Fira/archive/4.106.tar.gz
-outfile2=$(shell basename ${srcfile2})
-
+ARCHIVE		= master.zip
+ARCHIVE_LINK	= https://github.com/bBoxType/FiraSans/archive/$(ARCHIVE)
```

Comment:
What is this URL? 

I was careful to download from Mozilla having read their copyright.  Can we not just use a newer file from them?

Reply:
Never mind -- looked there now.